### ISSUE TITLE
Real space Eigenfunctions

### ISSUE THREAD

**Issue by shahid-sattar** [Z2PackDev/TBmodels#197]
Dear Developers,
Thanks for making the code available for users.

May I ask if it is possible to obtain real-space eigenfunctions from wannier90 output using TB models?

Thanks in advance!

**Comment by greschd** [comment]
No, this is not something that TBmodels can do natively. It takes the tight-binding form of the Hamiltonian as given, without keeping the connection to the underlying basis functions.

Could you maybe explain your use case a bit further? In general, this code can be used to parse the U matrices given by Wannier90: https://github.com/jimustafa/wannier90-utils
These connect the Wannier and DFT bases, so if you know the real-space eigenfunctions from the DFT calculations the appropriate basis transformation could be constructed.
